histomics_label/web_client/main.js

```diff
@@ -9,7 +9,7 @@ import './views/HeaderView';
 import './views/HeaderImageView';
 import * as WSISuperpixelGuidedLabeling from './index';
```

Comment:
Do we need to change `WSISuperpixelGuidedLabeling` to something in Lines 10 and 15?

Reply:
Good catch, thank you!